cmd/createorupdate/createorupdate.go

```diff
@@ -56,7 +50,7 @@ func createOrUpdate(oc *v1.OpenShiftCluster) (*v1.OpenShiftCluster, error) {
 
 	// validate the internal API representation (with reference to the previous
 	// internal API representation)
-	errs = p.ValidateInternal(cs, oldCs)
+	errs := p.ValidateInternal(cs, oldCs)
```

Comment:
Here is an example of logger that's passed by the service to ACS Engine: https://github.com/Azure/acs-engine/blob/master/pkg/operations/kubernetesupgrade/upgrader.go#L45. It's from: `github.com/sirupsen/logrus`

Reply:
:+1: I've created a card for this since it applies to all of the methods.  Will work on that this sprint

Reply:
all good on the removal of `ValidateExternal` that this introduces? https://github.com/openshift/openshift-azure/pull/155/files#diff-866712c7c0e6cc75c39b753f2976c400L11

Reply:
+1 on the use of logrus. We should import it and change it across all the codebase, so it needs to be a separate PR.

Reply:
+1 For logrus :D my favorite one